assets/src/setup/pages/welcome/index.js

```diff
@@ -0,0 +1,151 @@
+						</g>
+						<defs>
+							<clipPath id="clip0">
+								<rect width="199.4" height="150.9" fill="white" transform="translate(0.953064 0.601562)" />
+							</clipPath>
+						</defs>
+					</svg>
+				</div>
+				<h1>
+					{ __( 'Welcome to the Official AMP Plugin for WordPress', 'amp' ) }
+				</h1>
+			</div>
+			<div className="welcome__body">
+
+				<div className={ 'welcome__section' }>
+					<div className="welcome__section-icon">
+						<svg width="41" height="41" viewBox="0 0 41 41" fill="none" xmlns="http://www.w3.org/2000/svg">
+							<g clipPath="url(#clip0)">
+								<path d="M8.37808 39.1765C7.85308 39.1765 7.41558 38.739 7.41558 38.214C7.41558 38.1265 7.41558 38.1265 7.41558 38.039L9.60308 25.439L0.415576 16.514C0.0655762 16.164 0.0655762 15.5515 0.415576 15.114C0.590576 14.939 0.765576 14.8515 0.940576 14.8515L13.6281 13.014L19.3156 1.55146C19.5781 1.11396 20.1906 0.93896 20.7156 1.20146C20.8031 1.28896 20.8906 1.37646 20.9781 1.55146L26.6656 13.014L39.3531 14.8515C39.8781 14.939 40.2281 15.3765 40.1406 15.9015C40.1406 16.0765 40.0531 16.339 39.8781 16.4265L30.6906 25.439L32.8781 38.039C32.9656 38.564 32.6156 39.089 32.0906 39.1765C31.9156 39.1765 31.6531 39.1765 31.4781 39.089L20.1031 33.139L8.72808 39.089C8.64058 39.1765 8.46558 39.1765 8.37808 39.1765ZM3.12808 16.5146L11.2656 24.4771C11.5281 24.7396 11.6156 25.0021 11.5281 25.3521L9.60308 36.5521L19.6656 31.3021C19.9281 31.1271 20.2781 31.1271 20.5406 31.3021L30.6031 36.5521L28.7656 25.2646C28.6781 24.9146 28.8531 24.6521 29.0281 24.3896L37.1656 16.4271L25.8781 14.7646C25.5281 14.6771 25.2656 14.5021 25.1781 14.2396L20.1031 4.08958L15.0281 14.3271C14.8531 14.5896 14.5906 14.7646 14.3281 14.8521L3.12808 16.5146Z" fill="#005AF0" />
+								<path d="M18.8774 29.0242H17.9149L18.8774 23.3367H15.9899C15.7274 23.3367 15.5524 23.1617 15.5524 22.8992C15.5524 22.8117 15.6399 22.7242 15.6399 22.6367L20.8024 13.8867H21.7649L20.8024 19.5742H23.7774C24.0399 19.5742 24.2149 19.7492 24.2149 20.0117C24.2149 20.0992 24.2149 20.1867 24.1274 20.2742L18.8774 29.0242Z" fill="#005AF0" />
+							</g>
+							<defs>
+								<clipPath id="clip0">
+									<rect width="40" height="40" fill="white" transform="translate(0.153076 0.125)" />
+								</clipPath>
+							</defs>
+						</svg>
+
+					</div>
+					<div className="welcome__section-description">
+						<h4>
+							{ __( 'AMP and WordPress', 'amp' ) }
+						</h4>
+						<p dangerouslySetInnerHTML={ { __html: __( 'AMP is a web framework designed to make it easier to build user-first sites--that is, sites that are beautiful, fast, engaging, secure, and accessible. <a href="http://amp.dev" target="_blank" rel="noreferrer">Learn more.</a> The Official AMP plugin for WordPress enables you to incorporate AMP into your site.', 'amp' ) } } />
```

Comment:
I don't think this will be a easily translatable string, especially with HTML being embedded.  Why not something like:

```suggestion
						<p>
							{ __( 'AMP is a web framework designed to make it easier to build user-first sites--that is, sites that are beautiful, fast, engaging, secure, and accessible.', 'amp' ) }
							{ ' ' }
							<a href="http://amp.dev" target="_blank" rel="noreferrer">
								{ __( 'Learn more', 'amp' ) }
							</a>
							{ '. ' }
							{ __( 'The Official AMP plugin for WordPress enables you to incorporate AMP into your site.', 'amp' ) }
						</p>
```